tests/rptest/tests/partition_movement_test.py

```diff
@@ -439,8 +439,12 @@ def partition_ready():
 
         # Another move should fail
         assert admin.get_partitions(name, 0)['status'] == "in_progress"
-        r = admin.set_partition_replicas(name, 0, old_assignments)
-        assert r.status_code == 400
+        try:
+            r = admin.set_partition_replicas(name, 0, old_assignments)
+        except requests.exceptions.HTTPError as e:
```

Comment:
When did this behavior change? Quick look at [latest requests doc](https://docs.python-requests.org/en/latest/api/) doesn't mention it raising an exception.

Reply:
Changed in https://github.com/vectorizedio/redpanda/pull/2407 -- requests doesn't raise by default but we now do.  Otherwise too easy for test authors to ignore results.